array/builder.go

```diff
@@ -1,6 +1,7 @@
 package array
 
 import (
+	"github.com/apache/arrow/go/arrow/bitutil"
```

Comment:
Curious if you needed to import this module without the `v7`?

Reply:
That was just autogenerated via the LSP , fixe